### ISSUE TITLE
what kind of captioner to use?

### ISSUE THREAD

**Issue by dongdk** [showlab/PhotoDoodle#7]
hi what kind of captioner do u use to caption these pair images?

**Comment by songyiren725** [comment]
During the pre-training phase, we used captions from the [[SEED-Data-Edit](https://huggingface.co/datasets/AILab-CVC/SEED-Data-Edit)](https://huggingface.co/datasets/AILab-CVC/SEED-Data-Edit) dataset. For the EditLoRA training dataset, due to the limited number of samples and the requirement for an instruction-based format, we manually annotated the data.
